docs/source/faq.rst

```diff
@@ -15,6 +15,15 @@ for larger modules.
 __ http://stackoverflow.com/questions/39295200/can-i-use-functions-imported-from-py-files-in-dask-distributed
 .. _API docs: http://distributed.readthedocs.io/en/latest/api.html#distributed.executor.Executor.upload_file
 
+Can I launch a tasks from a task?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This could be while averaging inside another function submitted with
+dask.distributed. For more detail and explanation, see
```

Comment:
Maybe swapping the order of "build understanding" and "additional features" so "additional features" is on top? This looks like it makes sense for what those two sections cover too.

And maybe include a link in the FAQ pointing to "additional features"?